src/person/utils.py

```diff
@@ -0,0 +1,35 @@
+from api.settings import IMAGE_BUCKET_NAME
+from api.settings import IMAGE_UPLOAD_URL
+from django.contrib.auth.models import User
+import requests
+
+
+def upload_profile_pic(user_id, profile_pic_base64):
+    """Uploads image to server and modifies user's profile_pic_url"""
+    request_body = {
+        "bucket": IMAGE_BUCKET_NAME,
+        "image": profile_pic_base64,
+    }
+    response = requests.post(IMAGE_UPLOAD_URL + "upload/", json=request_body)
+    if response.status_code != 201:
+        return
+    user = User.objects.get(id=user_id)
+    person = user.person
+    if person.profile_pic_url:
```

Comment:
If there is already a profile picture, make sure to delete the old one to free up space on the server. Add a comment like this.